src/concepts/workflows.md

```diff
@@ -1,82 +1,27 @@
 # Workflows
 
-Workflows are the main way to customize Caido to suit your needs.
-A workflow consists of three components:
+## What are Workflows?
 
-- [Nodes](#nodes)
-- [Connections](#connections)
-- [Inputs & Outputs](#inputs--outputs)
+---
 
-<img alt="Convert workflow" src="/_images/workflow_convert_basic.png" center/>
-
-## Nodes
-
-Nodes are draggable blocks that perform specific actions. Add nodes to your workflow by dragging blocks from the left panel.
-
-<img alt="List of nodes" src="/_images/workflow_convert_node_list.png" height="500" center/>
-
-Clicking a node reveals its properties.
-A node takes a set of `inputs` and optionally produces a set of `outputs`.
-Each node has a customizable `alias` which is used to uniquely identify itself inside a workflow.
-
-<img alt="Base64 Node" src="/_images/workflow_convert_node_base64.png" width="500" center/>
-
-Caido comes with a default list of nodes. Here are a few we support:
-
-- Encoding nodes (Base64/URL/Hex)
-- Hashing nodes (SHA1/SHA2)
-- Code nodes (Shell/JavaScript)
-- Control nodes (If-Else/If-Else-Javascript)
-- Misc nodes (Set-Color/HTTPQL)
+**Workflows** are the main way to customize Caido to suit your needs. Think of Workflows as custom shortcuts that will perform an action/a sequence of actions under certain specified conditions. These "shortcuts" can be saved as a Workflow in Caido so they can be utilized repeatedly.
```

Comment:
It is not the main way anymore, its A way to customize caido with tasks executed on the server side (ref to client/server architecture).
Unsure about the `shortcuts` analogy, we generally define them as building your own mini-plugins by combining multiple actions and scripts together (`that will perform an action/a sequence of actions under certain specified conditions` that is good).